_data-prepper/pipelines/pipelines.md

```diff
@@ -84,269 +88,8 @@ conditional-routing-sample-pipeline:
-
-```bash
-docker run --name data-prepper \
-    -v /full/path/to/logstash.conf:/usr/share/data-prepper/pipelines/pipelines.conf \
-    opensearchproject/opensearch-data-prepper:latest
-```
-
-This feature is limited by feature parity of Data Prepper. As of Data Prepper 1.2 release, the following plugins from the Logstash configuration are supported:
-
-- HTTP Input plugin
-- Grok Filter plugin
-- Elasticsearch Output plugin
-- Amazon Elasticsearch Output plugin
-
-## Configure the Data Prepper server
-
-Data Prepper itself provides administrative HTTP endpoints such as `/list` to list pipelines and `/metrics/prometheus` to provide Prometheus-compatible metrics data. The port that has these endpoints has a TLS configuration and is specified by a separate YAML file. By default, these endpoints are secured by Data Prepper docker images. We strongly recommend providing your own configuration file for securing production environments. Here is an example `data-prepper-config.yaml`:
-
-```yml
-ssl: true
-keyStoreFilePath: "/usr/share/data-prepper/keystore.jks"
-keyStorePassword: "password"
-privateKeyPassword: "other_password"
-serverPort: 1234
-```
-
-To configure the Data Prepper server, run Data Prepper with the additional yaml file.
-
-```bash
-docker run --name data-prepper \
-    -v /full/path/to/my-pipelines.yaml:/usr/share/data-prepper/pipelines/my-pipelines.yaml \
-    -v /full/path/to/data-prepper-config.yaml:/usr/share/data-prepper/data-prepper-config.yaml \
-    opensearchproject/data-prepper:latest
-```
-
-## Configure peer forwarder
-
-Data Prepper provides an HTTP service to forward events between Data Prepper nodes for aggregation. This is required for operating Data Prepper in a clustered deployment. Currently, peer forwarding is supported in `aggregate`, `service_map_stateful`, and `otel_trace_raw` processors. Peer forwarder groups events based on the identification keys provided by the processors. For `service_map_stateful` and `otel_trace_raw` it's `traceId` by default and can not be configured. For `aggregate` processor, it is configurable using `identification_keys` option. 
-
-Peer forwarder supports peer discovery through one of three options: a static list, a DNS record lookup , or AWS Cloud Map. Peer discovery can be configured using `discovery_mode` option. Peer forwarder also supports SSL for verification and encryption, and mTLS for mutual authentication in a peer forwarding service.
-
-To configure peer forwarder, add configuration options to `data-prepper-config.yaml` mentioned in the [Configure the Data Prepper server](#configure-the-data-prepper-server) section:
-
-```yml
-peer_forwarder:
-  discovery_mode: dns
-  domain_name: "data-prepper-cluster.my-domain.net"
-  ssl: true
-  ssl_certificate_file: "<cert-file-path>"
-  ssl_key_file: "<private-key-file-path>"
-  authentication:
-    mutual_tls:
-```
-
-
-## Pipeline Configurations
```

Comment:
Deleted this example because it's redundant to https://opensearch.org/docs/latest/data-prepper/pipelines/pipelines/